src/screens/Welcome/WelcomeScreen.tsx

```diff
@@ -1,26 +1,32 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
 import { PageContainer } from 'screens/styles';
+import { Image } from 'react-native';
 import { TextRegular, TextBold, TextRegularWhite } from 'assets/fonts/Fonts';
 import { ButtonDark, ButtonLight } from 'assets/Components';
 import {
   ContentContainer,
   TitleContainer,
   ButtonContainer,
   ButtonView,
+  ImageStyles,
 } from './styles';
 
```

Comment:
i removed it from the file but just make sure to remove any comments that don't provide functionality before creating your final pr